catalog/admin/module/catalog-admin-module-sources/src/main/webapp/index.html

```diff
@@ -33,6 +33,9 @@
     <![endif]-->
 </head>
 <body>
+<div id="page-loading">
```

Comment:
You can move this into the main container and it will be overwritten when Backbone is ready to render the view.